scripts/create-kubestellar-demo-env.sh

```diff
@@ -17,16 +17,27 @@
 
 set -e
 
-kubestellar_version=0.25.0-rc.1
+export kubestellar_version=0.25.0-rc.1
```

Comment:
Why the "export"? If it is not needed then I think that it is positively harmful to include it because that makes the reader unsure of whether any invoked executable depends on that variable.

Reply:
can be removed I guess

Reply:
Removing it would be my recommendation.

Reply:
removed